.github/workflows/ci.yml

```diff
@@ -23,6 +23,8 @@ jobs:
           fetch-depth: 0
 
       - uses: pnpm/action-setup@v2.2.4
+        with:
+          version: 8
```

Comment:
pnpm 8 is different from the configured version in package.json, need to update there with a specific (8.x.x) version and lock file as well.